frontend/src/components/SVGDisplay.jsx

```diff
@@ -44,7 +44,7 @@ export const SVGDisplay = ({ content }) => {
         {content && <div dangerouslySetInnerHTML={{ __html: content }}></div>}
       </div>
       {showLightbox && (
-        <Lightbox
+        <Lightbox id="svg-lightbox"
```

Comment:
What's the ID here used for? (Not for the CSS above, that selector looks at classes.)